### DISCUSSION TITLE
a beginners question about quasar

### DISCUSSION THREAD

**Discussion by ghost** [quasarframework/quasar#9845] [General - Components / Directives / etc]
I'm having problems creating an minesweeper for my assignment, due to the fact that everytime that I try to build the js. code on the MainLayout it doesn't work. I don't really know how to use the framework.

I'm watching this video to build/learn the code: https://www.youtube.com/watch?v=W0No1JDc6vE&t=312s

**Comment by MilosPaunovic** [comment]
Hello @rc-rodrigues,

Firstly, I would like to point you to the official documentation for the [Quasar 2](https://quasar.dev/start/pick-quasar-flavour) or [Quasar 1](https://v1.quasar.dev/), depending on which one you are using. It's very extensive and beginner friendly.

If you could be more specific about the problem you're having, I could try to help you out.

**Reply by ghost** [reply]
So, I received an assignment asking for me to build an Minesweeper game, mostly using div, on Quasar with Vue. I looked for an tutorial and found the one I linked before, but it doesn't seem to work. Writing the whole code from the tutorial on the MainLayout doesn't work, because somehow the javascript seems to not be accepted, but simple commands like alert do.

I received an exemple of javascript written in the quasar framework, and the 'methods' are apart, should I try the same? Or perhaps I just have to put the code in other document?

**Reply by MilosPaunovic** [reply]
Without any code snippet I'll hardly be able to assist you, but it seems like you have issues with Vue syntax itself - maybe you could check out some more of their [documentation](https://vuejs.org/v2/guide/index.html) and that might help with understanding your assignment better.
